tony-history-server/app/controllers/HomeController.java

```diff
@@ -0,0 +1,102 @@
+package controllers;
+
+import com.typesafe.config.Config;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import javax.inject.Inject;
+
+import org.apache.hadoop.fs.FSDataInputStream;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hdfs.HdfsConfiguration;
+import org.apache.hadoop.security.UserGroupInformation;
+import org.apache.hadoop.yarn.api.ApplicationConstants;
+import org.apache.hadoop.yarn.conf.YarnConfiguration;
+
+import play.Logger.ALogger;
+import play.Logger;
+import play.mvc.*;
+
+
+public class HomeController extends Controller {
```

Comment:
Didn't get a chance to reply to previous conversation, i'd prefer something like TonyHomePageController - this controller is for the homepage of tony

Reply:
That sounds good to me.

Reply:
I think right now, we don't have enough views/controllers where naming becomes a problem just yet. We can iterate and refactor later as the app gets more complex.